test/osal-windows.cc

```diff
@@ -186,10 +186,10 @@ bool actor_config::osal_deserialize(const char *str, const char *end,
 //-----------------------------------------------------------------------------
 
 typedef std::pair<HANDLE, actor_status> child;
-static std::unordered_map<mdbx_pid_t, child> childs;
+static std::unordered_map<mdbx_pid_t, child> children;
```

Comment:
I'm not uncertain about this

Reply:
@jsoref, thank you very much.
I will definitely accept this PR.

There are indeed inaccuracies in the corrections, so I will have to carefully review the proposed changes.
Some of the inaccuracies are due to the use of jargon, bad names, and abbreviations.
Nonetheless most of these cases are now clearly visible and there is a reason to correct everything.

In the near future, I will try to apply your changes, but in the devel branch, so as not to cause unnecessary difficulties during the subsequent merge.

Thanks again.
Regards.